Created (kanienkeha) aan  (Natural Speech) and aae (edu/slow speech)

dlothian opened the pull request:
### Type of Pull Request

Add tags for aan (kanienkeha natural speech) and aae (educational speech)

- [x] Bulk sentence upload 

<!--- Please ensure your sentences are licensed under CC0 before making a submission. Learn how you can do this via our Playbook https://common-voice.github.io/community-playbook/sub_pages/cc0waiver_process.html -->

- [ ] Related to a listed issue 

<!--- Please link the issues related to your PR Request -->

*
*

- [ ] Other

<!--- Please describe the pull request-->

*
*

### Acknowledging contributors

<!-- Please list who collaborated with you to make this contribution. Or Community dicussion that helped make this idea possible -->

*
*

dlothian committed:
feat(aae,ann): Kanien’kéha (Kahnawa’kéha) Educational and Natural Speech

dlothian committed:
feat(aae,ann): Kanien’kéha (Kahnawa’kéha) Educational and Natural Speech

dlothian committed:
Names of aae and aan

dlothian committed:
aae + ann, kanienkeha edu and natural speech

dlothian committed:
feat (aae, aan): kanienkeha edu/natural tags

dlothian committed:
Create cross-locale.ftl for aae

dlothian committed:
Create messages.ftl for aae

dlothian committed:
Create messages.ftl for aan

dlothian committed:
Create cross-locale.ftl for aan

Commit:
updating submodules

SamuelLarkin reviewed (changes requested):
Thanks for keep stuff in alphabetical order.
Probably the instructions weren't clear enough on this but you actually also have to update
all `web/locales/*/messages.ftl` for all locales because otherwise, the common-voice webpage will be missing strings.  By that I mean, if you only update `en`, as you did, then viewing the site where you tell it you are English, then you see all the strings but if you then switch to view the site as a `moh` speaker for example, then you will be missing the new strings.

dlothian committed:
added aae and aan

dlothian committed:
added aae and aan

dlothian committed:
added aae and aan

dlothian committed:
added aae and aan to the rest of the messages.ftl file

dlothian committed:
Script to automate adding a new language code

Manually specify an iso and description. Will avoid adding duplicate isos, but will not check if its a valid iso.

dlothian commented:
Hey sam, I finished what you requested.

I hadnt cloned it to my computer so I started doing it one at a time, which is obviously bad practice so I stopped and switched to my local machine. While I was doing that I thought it would be useful to have an executable to help with this, and then I accidentally wrote a python script to do it with your instructions as the comments. 

It's not amazing script because I threw it together quickly, but it definitely works. You have to specify the iso and description in a dictionary and then it does the rest. It will not add it if it already exists in the files, but it will try all files it should be in.

SamuelLarkin commented on add_new_language.py:
```diff
@@ -0,0 +1,90 @@
+
+
+def copyfile(iso, filename):
+    if not os.path.isfile("web/locales/{}/{}".format(iso,filename)):
```
I think you could've used `shutil.copyfile()`.

SamuelLarkin commented on add_new_language.py:
```diff
@@ -0,0 +1,90 @@
+            locales.append(iso)
+        else:
+            locales[iso] = des
+        json.dump(locales,open(filename, "w"),indent=2)
```
I think we should add `ensure_ascii=False` otherwise
```diff
-  "bn": "বাংলা",
+  "bn": "\u09ac\u09be\u0982\u09b2\u09be",
```
which makes the files far less human readable.

SamuelLarkin replied:
Can we also keep the keys sorted
```python
json.dump(sorted(locales), open(filename, "w"), indent=2, ensure_ascii=False)
```

dlothian replied:
Oh weird, I had ensure_ascii=False at one point, it must have floated off into the ether when moving things around. Also yes sorted sounds good.

SamuelLarkin reviewed (changes requested):
two minor changes, sorted list and ensure_ascii=False

SamuelLarkin commented:
Great idea to make a script to automate adding a new locale.

dlothian committed:
fix:requested updates

ensure ascii
sorted lists
shutil copy file

dlothian commented:
Ok, requested changes have been added!

SamuelLarkin reviewed (approved):
Sorry for nitpicking 😃 on the sorting of locales.
